package.json

```diff
@@ -51,7 +51,8 @@
     "react-tools": "latest",
     "stylus": "latest",
     "swig": "latest",
-    "toffee": "latest"
+    "toffee": "latest",
+    "dot": "latest"
```

Comment:
if you run [fixpack](https://www.npmjs.com/package/fixpack) on this, it'll automatically alphabetize it for you... I think `npm install dot --save-dev` will do the same.